Changelog.md

```diff
@@ -19,6 +19,7 @@ Breaking Changes:
  * Type System: Disallow ``type(super)``.
  * Type System: Disallow enums with more than 256 members.
  * Type System: Disallow explicit conversions from negative literals and literals larger than ``type(uint160).max`` to ``address`` type.
+ * Type System: Introduce ``address(...).code`` to retrieve the code as ``bytes memory``. As expected the size can be obtained via ``address(...).code.length``.
```

Comment:
But there is no shortcut currently, is there?

Reply:
There is not shortcut. That can be done in a non-breaking release. Although my whole point in the original issue was to have the ability to query the codesize (and not retrieve the actual code).

Reply:
Do we want to explain that just retrieving the code using `address(...).code.length` also copies the full code (at least for now)?

Reply:
Yes, definitely :)